_FIPS201/requirements.md

```diff
@@ -49,7 +49,7 @@ Each agency's PIV implementation **SHALL** meet the control objectives listed ab
 + An issued credential is not duplicated or forged.
 + An issued credential is not modified by an unauthorized entity.
 
-[^initiation]: The initiation of a background investigation is defined as the submission of an investigative request to the Defense Counterintelligence and Security Agency or other authorized federal investigative service provider.
+[^initiation]: See [CSP]](../references/#ref-CSP) for investigation initiation process.
```

Comment:
Seems incomplete ... Details about? Information on?

Reply:
The commit has the explanation.  Or can add it here too.  Essentially, 'initiation' is  not enough. The service provider has to have scheduled interview as well, as per CSP.

Reply:
What I was saying is that this is more of a wording thing, like:

```suggestion
[^initiation]: See [CSP]](../references/#ref-CSP) for details about the investigation initiation process.
```